include/aws/http/connection.h

```diff
@@ -407,6 +407,16 @@ struct aws_http_client_connection_options {
      * Host resolution override that allows the user to override DNS behavior for this particular connection.
      */
     const struct aws_host_resolution_config *host_resolution_config;
+
+    /**
+     * Optional.
+     * MUST be larger than 0.
+     * If the time between the request finishes to be sent and the first byte of the response received larger than the
+     * set timeout, connection will be closed.
+     * Applies to all the request made from the connection.
+     * TODO: Only supported in HTTP/1.1 now, support it in HTTP/2
+     */
```

Comment:
```suggestion
    /**
     * Optional (ignored if 0).
     * After a request is fully sent, if the server does not begin responding within N milliseconds,
     * then fail with AWS_ERROR_HTTP_REQUEST_IDLE_TIMEOUT.
     * This can be overridden per-request by aws_http_make_request_options.idle_request_timeout.
     * TODO: Only supported in HTTP/1.1 now, support it in HTTP/2
     */
```